multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/Messages.java

```diff
@@ -202,6 +202,7 @@ public class Messages {
     public static final String COULD_NOT_BIND_OPTIONAL_SERVICE_TO_APP = "Could not bind optional service \"{0}\" to application \"{1}\"";
     public static final String COULD_NOT_UNBIND_OPTIONAL_SERVICE_TO_APP = "Could not unbind optional service \"{0}\" to application \"{1}\"";
     public static final String COULD_NOT_CREATE_OPTIONAL_SERVICE = "Could not create optional service \"{0}\"";
+    public static final String COULD_NOT_UPDATE_OPTIONAL_SERVICE = "Could not update optional service \"{0}\" {1}";
```

Comment:
Maybe reformat the message to be worded more clearly, like: 'Could not update **metadata** of option service "service-name"'